relayer/src/config.rs

```diff
@@ -110,6 +110,8 @@ pub struct Config {
     #[serde(default)]
     pub global: GlobalConfig,
     #[serde(default)]
+    pub rest: RESTConfig,
```

Comment:
Maybe we we can follow Rust conventions and call it `RestConfig` instead of `RESTConfig`,

Reply:
https://github.com/informalsystems/ibc-rs/pull/955/commits/555469eff1086e0541fb8550f9cad6ec0e4534f7